packages/Manager/src/main/resources/db/migration/R__setup_permissions.sql

```diff
@@ -0,0 +1,30 @@
+ -- Permissions for the Application user.
+GRANT
+  USAGE ON SCHEMA public TO eop_manager_app_user;
+
+ALTER DEFAULT PRIVILEGES
+  IN SCHEMA public
+GRANT
+SELECT
+,
+  INSERT,
+UPDATE
+,
+  DELETE ON TABLES TO eop_manager_app_user;
+
+GRANT
+SELECT
+,
+  INSERT,
+UPDATE
+,
+  DELETE ON ALL TABLES IN SCHEMA public TO eop_manager_app_user;
+
+-- Permissions for developer users debugging.
+GRANT
+  USAGE ON SCHEMA public TO developers;
```

Comment:
Do we also need a `ALTER DEFAULT PRIVILEGES` statement for the developers group?

Reply:
> Curious why the port and name are in here? Does that suggest we should also include host? Also, are the key names recognized automatically by Spring & Flyway once imported?

This was previously overriding this setting in `application.yml` ... 
```
  datasource:
    url: jdbc:postgresql://${CONFIG_DATABASE_HOST:localhost}:${CONFIG_DATABASE_PORT:5432}/${CONFIG_DATABASE_NAME:eop_dev}
    username: ${CONFIG_DATABASE_USERNAME:eop_manager_app_user}
    password: ${CONFIG_DATABASE_PASSWORD:password}
```

And needed to use different names because that the names Gruntworks had used in their secret (called RDSDBConfig) ... now we've created a new secret just for the application, it uses the same naming as application.yml expects